packages/experimental/Switch/src/useSwitch.ts

```diff
@@ -64,3 +89,11 @@ export const useSwitch = (props: SwitchProps): SwitchInfo => {
     },
   };
 };
+
+const getAccessibilityState = memoize(getAccessibilityStateWorker);
+function getAccessibilityStateWorker(checked: boolean, disabled: boolean, accessibilityState?: AccessibilityState) {
+  if (accessibilityState) {
+    return { checked, disabled, ...accessibilityState };
```

Comment:
I think this needs to be specified as `checked: checked, disabled: disabled, ...` (same as line 98).

Reply:
I wrote it like that because that's how I saw other files like `useCheckbox.ts` do it. It looks like it's just a syntatic shortcut for when the variable's name and the key's name are the same. Here's a [article](https://attacomsian.com/blog/javascript-object-property-shorthand#:~:text=The%20object%20literal%20property%20value,make%20the%20object%20initialization%20shorter.&text=The%20property%20value%20shorthand%20syntax,value%20as%20a%20property%20value.) with an example. I can still change it if you'd like though.

Reply:
Yeah, I'm also interested why we can't use shorten version. Is it some convention that we need to write `checked: checked`?

Reply:
Oh interesting, I didn't realize that worked. That's totally fine then. Thanks for teaching me something new 🙂